runtime/src/main/java/io/quarkus/vault/runtime/client/secretengine/VaultInternalTransitSecretEngine.java

```diff
@@ -1,83 +1,89 @@
 package io.quarkus.vault.runtime.client.secretengine;
 
-import jakarta.inject.Singleton;
-
 import io.quarkus.vault.runtime.client.VaultClient;
 import io.quarkus.vault.runtime.client.VaultInternalBase;
-import io.quarkus.vault.runtime.client.dto.transit.VaultTransitCreateKeyBody;
-import io.quarkus.vault.runtime.client.dto.transit.VaultTransitDecrypt;
-import io.quarkus.vault.runtime.client.dto.transit.VaultTransitDecryptBody;
-import io.quarkus.vault.runtime.client.dto.transit.VaultTransitEncrypt;
-import io.quarkus.vault.runtime.client.dto.transit.VaultTransitEncryptBody;
-import io.quarkus.vault.runtime.client.dto.transit.VaultTransitKeyConfigBody;
-import io.quarkus.vault.runtime.client.dto.transit.VaultTransitKeyExport;
-import io.quarkus.vault.runtime.client.dto.transit.VaultTransitListKeysResult;
-import io.quarkus.vault.runtime.client.dto.transit.VaultTransitReadKeyResult;
-import io.quarkus.vault.runtime.client.dto.transit.VaultTransitRewrapBody;
-import io.quarkus.vault.runtime.client.dto.transit.VaultTransitSign;
-import io.quarkus.vault.runtime.client.dto.transit.VaultTransitSignBody;
-import io.quarkus.vault.runtime.client.dto.transit.VaultTransitVerify;
-import io.quarkus.vault.runtime.client.dto.transit.VaultTransitVerifyBody;
+import io.quarkus.vault.runtime.client.dto.transit.*;
 import io.smallrye.mutiny.Uni;
+import jakarta.inject.Singleton;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
 
 @Singleton
 public class VaultInternalTransitSecretEngine extends VaultInternalBase {
 
+    /**
+     * Concatenates the provided URL path segments and sanitizes separators.
+     * The path is returned without a leading "/".
+     *
+     * @param firstPart first URL path segment
+     * @param otherParts other URL path segments
+     * @return String concatenated path
+     */
+    private String getPath(String firstPart, String... otherParts) {
```

Comment:
Wouldn't handling nulls here cleanup the call sites even more?

Reply:
In practice, the parameters should not be able to be null. But I have added null handling anyway 👍